# =============== INITIAL JOINT STATE ===============
JOINT_1: 2.912
JOINT_2: 0.427
JOINT_3: 0.665
JOINT_4: 1.262
JOINT_5: -0.323
JOINT_6: 0.943
JOINT_7: 5.240

# =============== ROBOT PARAMETERS ===============
GAIN: 2
SAMPLING_TIME: 0.01

# =============== MANIPULABILITY PARAMETERS (NOT USED) ===============
ENABLE_MANIPULABILITY: false
DERIVATIVE_STEP: 0.00001
NULL_GAIN: 20

# =============== PLANNER PARAMETERS ===============
TIME_TO_GRAB: 60
TIME_TO_RELEASE: 60
TIME_DELAY: 0.2
APPROACH_DISTANCE: 0.07
ZONE1_THRESHOLD: 0.3
ZONE2_THRESHOLD: 0.6